.github/workflows/publish-docs.yaml

```diff
@@ -0,0 +1,115 @@
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+
+      - name: Install Deps
+        run: |
+          sudo apt update -y
+          DEBIAN_FRONTEND=noninteractive sudo apt install -y git cmake doxygen graphviz python3-pip python3-virtualenv python3-setuptools python3-stemmer wget
+
+      - uses: actions/cache@v1
+        id: sphinx-cache
+        with:
+          path: /home/runner/python-env/sphinx
+          key: ${{ runner.os }}-${{ hashFiles('**/sphinx-lockfiles') }}
+
+      - name: Install Sphinx
+        if: steps.cache.outputs.sphinx-cache-hit != 'true'
+        run: |
+          python3 -m venv /home/runner/python-env/sphinx
+          . /home/runner/python-env/sphinx/bin/activate
+          python3 -m pip install sphinx
+          python3 -m pip install myst-parser
+          python3 -m pip install sphinx-rtd-theme
+          python3 -m pip install rinohtype
+          python3 -m pip install pillow
+          python3 -m pip install gitpython
+          python3 -m pip install breathe
+          python3 -m pip install pygments
+
+      - uses: erlef/setup-beam@v1
+        with:
+          otp-version: "24"
+          elixir-version: "1.14"
+
+      - name: Install rebar3
+        working-directory: /tmp
+        run: |
+          wget https://s3.amazonaws.com/rebar3/rebar3 && chmod +x rebar3
+          ./rebar3 local install
+          echo "/home/runner/.cache/rebar3/bin" >> ${GITHUB_PATH}
+
+      - name: "Checkout repo"
+        uses: actions/checkout@v2
+        with:
+          submodules: 'recursive'
+
+      - name: Get branch name
+        id: get_branch
+        run: echo ::set-output name=short_ref::${GITHUB_REF#refs/*/}
+
+      - uses: erlef/setup-beam@v1
+        with:
+          otp-version: "24"
+          elixir-version: "1.14"
+
+      - uses: actions/checkout@v3
+        id: checkout-production
+        with:
+          repository: UncleGrumpy/atomvm_www
+          ref: Production
+          path: /home/runner/work/AtomVM/AtomVM/www
+
+      - name: Build Site
+        run: |
+          . /home/runner/python-env/sphinx/bin/activate
+          mkdir build
+          cd build
+          cmake ..
+          cd doc
+          make GitHub_CI_Publish_Docs
+
+      - name: Commit files
+        working-directory: /home/runner/work/AtomVM/AtomVM/www
+        run: |
+          git config --local user.email "atomvm-doc-bot@object.stream"
+          git config --local user.name "AtomVM Doc Bot"
+          git add .
+          git commit -m "Update Documentation"
+      - name: Push changes
+        working-directory: /home/runner/work/AtomVM/AtomVM/www
+        run: |
+          eval `ssh-agent -t 60 -s`
+          echo "${{ secrets.PUBLISH_ACTION_KEY }}" | ssh-add -
+          mkdir -p ~/.ssh/
+          ssh-keyscan github.com >> ~/.ssh/known_hosts
+          git remote add push_dest "git@github.com:UncleGrumpy/atomvm_www.git"
```

Comment:
Should this be git@github.com:atomvm/atomvm_www.git ?

Reply:
It will be, when the final PR is merged... The last commit will be dropped - I had to push temporary changes to make the demonstration work.